{"links": {"self": {"href": "data/repositories/osrf/sdformat/issues/29/comments/7068209.json"}, "html": {"href": "#!/osrf/sdformat/issues/29#comment-7068209"}}, "issue": {"links": {"self": {"href": "data/repositories/osrf/sdformat/issues/29.json"}}, "type": "issue", "id": 29, "repository": {"links": {"self": {"href": "data/repositories/osrf/sdformat.json"}, "html": {"href": "#!/osrf/sdformat"}, "avatar": {"href": "data/bytebucket.org/ravatar/{b6d52f9f-b070-41c0-807c-94af07ea375b}ts=1606789"}}, "type": "repository", "name": "sdformat", "full_name": "osrf/sdformat", "uuid": "{b6d52f9f-b070-41c0-807c-94af07ea375b}"}, "title": "incompatibility between sdformat's URDF and github's URDF"}, "content": {"raw": "I just merged sdf_1.4 into default in 96181e5, so that should get the fix into the next nightly build. Thanks for the reminder @po1 ", "markup": "markdown", "html": "<p>I just merged sdf_1.4 into default in <a href=\"#!/osrf/sdformat/commits/96181e5\" rel=\"nofollow\" class=\"ap-connect-link\">96181e5</a>, so that should get the fix into the next nightly build. Thanks for the reminder @po1 </p>", "type": "rendered"}, "created_on": "2013-11-26T17:35:21.798017+00:00", "user": {"display_name": "Steve Peters", "uuid": "{2ccfed09-18b8-4921-8d58-15ef01092802}", "links": {"self": {"href": "https://api.bitbucket.org/2.0/users/%7B2ccfed09-18b8-4921-8d58-15ef01092802%7D"}, "html": {"href": "https://bitbucket.org/%7B2ccfed09-18b8-4921-8d58-15ef01092802%7D/"}, "avatar": {"href": "data/secure.gravatar.com/avatar/1fb4816dad9e68337d3096f750951f6cd=httpsavatar-management--avatars.us-west-2.prod.public.atl-paas.netinitialsSP-1.png"}}, "nickname": "Steven Peters", "type": "user", "account_id": "557058:5de38267-b118-494c-aa76-4fab35448816"}, "updated_on": null, "type": "issue_comment", "id": 7068209}